src/anchor.js

```diff
@@ -28,7 +29,43 @@ const aggregators = {
             r += part.r
         }
         return new Point(x / len, y / len, r / len)
-    }
+    },
+    intersect: (config, name, parts) => {
+        const get_line_from_point = (point, axis, offset=1000) => {
+            let p1 = [point.x - offset, point.y]
+            let p2 = [point.x + offset, point.y]
+            if(axis == 'y') {
+                p1 = [point.x, point.y - offset]
+                p2 = [point.x, point.y + offset]
+            }
+
+            let line = new m.paths.Line(p1, p2)
+            line = m.path.rotate(line, point.r, point.p)
+
+            return line
+        };
+
+        a.unexpected(
+            config, name,
+            [...aggregator_common, ...['axis1', 'axis2']]
+        )
+        a.in(config.axis1, 'axis1', ['x', 'y'])
+        a.in(config.axis2, 'axis2', ['x', 'y'])
+        a.arr(parts, 'parts', 2, 'Point')
+
+        const line1 = get_line_from_point(parts[0], config.axis1)
+        const line2 = get_line_from_point(parts[1], config.axis2)
+        var intersection = m.path.intersection(line1, line2);
```

Comment:
Nice job deferring to makerjs for the actual business logic - wouldn't want to reinvent the wheel :smile: Please use `let` instead of `var`, tho...